base/random.jl

```diff
@@ -1384,7 +1406,7 @@ function randexp_unlikely(rng, idx, x)
 end
 
 """
-    randn!([rng=GLOBAL_RNG], A::AbstractArray) -> A
+    randn!([rng::AbstractRNG=GLOBAL_RNG,] A::AbstractArray) -> A
```

Comment:
In that case, perhaps change the comma-bracket order of the signatures that you touch earlier in the file that follow the other order, and then at some future point we'll migrate away from the brackets for leading optional arguments? :)